lib/controllers/bikes.js

```diff
@@ -0,0 +1,29 @@
+const { Router } = require('express');
+const Bike = require('../models/Bike');
+const BikeService = require('../services/BikeService');
+
+module.exports = Router()
+  .post('/', async (req, res) => {
+    const bike = await BikeService.create(req.body);
```

Comment:
We should add `try / catch` 's here passing the error into `next` so that our errors are caught and handled. This will ensure that the entire API does not go down if an error occurs